app/views/dashboard/_past_applications.haml

```diff
@@ -5,7 +5,7 @@
       .note_item
         = select_tag :department_ids,
           options_from_collection_for_select(@departments, :id, :name),
-        class: 'dept_select', multiple: true
```

Comment:
OK so I deleted the files in vendor, because you're absolutely right and I forgot about rails-assets. I followed the instructions on the rails asset site for how to install a gem that way and it looked like everything worked when I ran bundle.. BUT. 
![screen shot 2016-01-25 at 4 11 26 pm](https://cloud.githubusercontent.com/assets/9645316/12564634/50058e16-c37e-11e5-9351-e48e09446c4b.png)
Did I miss a step? It said "installing rails-assets-multiple-select' and said bundle complete...